integration-tests/features/httpd-stateless.feature

```diff
@@ -1,14 +1,5 @@
 Feature: Redeploy stateless HTTP services
 
-Scenario: Migrate between remote hosts using virt-tar-out
-   Given the local virtual machines:
-         | name       | definition          | ensure_fresh |
-         | app-source | centos6-guest-httpd | no           |
-         | target     | centos7-target      | no           |
-    When app-source is migrated to target as a macrocontainer
```

Comment:
This simpler step definition can be reclaimed to refer to the rsync-based migration